driver/ppm_fillers.c

```diff
@@ -2409,6 +2434,26 @@ int f_sys_recv_x(struct event_filler_arguments *args)
 	return res;
 }
 
+int f_sys_recvfrom_e(struct event_filler_arguments *args)
+{
+	int res = 0;
+	unsigned long val = 0;
+	s32 fd = 0;
+
+	/* Parameter 1: fd (type: PT_FD) */
+	syscall_get_arguments_deprecated(current, args->regs, 0, 1, &val);
```

Comment:
I think this should consider `socketcall` here:

```suggestion
	if(!args->is_socketcall)
	{
		syscall_get_arguments_deprecated(current, args->regs, 0, 1, &val);
	}
#ifndef UDIG
	else
	{
		val = args->socketcall_args[0];
	}
#endif

```

Reply:
great catch ty :)